covid_graphs/covid_graphs/country_graph.py

```diff
@@ -24,17 +26,35 @@ def __str__(self):
         return self.value
 
 
+@dataclass
+class Trace:
+    xs: List[datetime.date]
+    ys: np.ndarray
+    x_max: datetime.date
+    y_max: float
```

Comment:
Note: I like @rejdi's naming for the rest server: `max_value` and `max_value_date`. Not completely convinced if suitable here but you can decide.

Reply:
Done